gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/FsFlowGraphMonitor.java

```diff
@@ -61,24 +63,29 @@ public class FsFlowGraphMonitor extends AbstractIdleService implements FlowGraph
   private final PathAlterationObserverScheduler pathAlterationDetector;
   private final FSPathAlterationFlowGraphListener listener;
   private final PathAlterationObserver observer;
-  private final Path flowGraphPath;
+  private Path flowGraphPath;
+  private Path observedPath;
   private final MultiHopFlowCompiler compiler;
   private final CountDownLatch initComplete;
   private static final Config DEFAULT_FALLBACK = ConfigFactory.parseMap(ImmutableMap.<String, Object>builder()
       .put(ConfigurationKeys.FLOWGRAPH_ABSOLUTE_DIR, DEFAULT_FS_FLOWGRAPH_MONITOR_ABSOLUTE_DIR)
       .put(ConfigurationKeys.FLOWGRAPH_BASE_DIR, DEFAULT_FS_FLOWGRAPH_MONITOR_FLOWGRAPH_DIR)
       .put(ConfigurationKeys.FLOWGRAPH_POLLING_INTERVAL, DEFAULT_FLOWGRAPH_POLLING_INTERVAL)
       .put(ConfigurationKeys.FLOWGRAPH_JAVA_PROPS_EXTENSIONS, ConfigurationKeys.DEFAULT_PROPERTIES_EXTENSIONS)
+      .put(MONITOR_TEMPLATE_CATALOG_CHANGES, false)
```

Comment:
why is this false by default? I thought we want this enabled to refresh from both together

Reply:
Going under the assumption that the flowgraph and the templates are not colocated in the same directory for other setups.